src/Features/Core/Portable/PullMemberUp/MembersPuller.cs

```diff
@@ -219,8 +230,9 @@ private static ISymbol GetSymbolsToPullUp(MemberAnalysisResult analysisResult)
                     eventSymbol,
                     accessibility: Accessibility.Public,
                     modifiers: modifiers);
-                var options = new CodeGenerationOptions(generateMethodBodies: false);
-                var publicAndNonStaticSyntax = codeGenerationService.CreateEventDeclaration(publicAndNonStaticSymbol, destination: CodeGenerationDestination.ClassType, options: options);
+
+                var eventGenerationOptions = options with { Context = new CodeGenerationContext(generateMethodBodies: false) };
```

Comment:
intresting.  why the 'with' here instead of just a new context guy?

Reply:
We need options with a new context and the current preferences.